### PR TITLE
Faux Fur survivor gear and recipes

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
Content "Adds Faux Fur survivor gear"

#### Purpose of change

Add's faux fur survivor gear. Addresses #47017

#### Describe the solution

Add's the items and recipes.

#### Describe alternatives you've considered

not adding them.

#### Testing

Need to change all the recipes from autolearn.

### CODE REVIEW

**Comment by SirPendrak** [comment]
Maybe they could use copy-from from regular winter surv. gear and just change materials and desciption?

File: data/json/items/armor/boots.json [hunk 1]
```diff
@@ -274,6 +274,28 @@
     "proportional": { "weight": 1.15, "volume": 1.15 },
     "flags": [ "VARSIZE", "WATERPROOF", "OVERSIZE" ]
   },
+  {
+    "id": "boots_wsurvivor_nofur",
+    "type": "ARMOR",
+    "name": { "str": "pair of faux fur winter survivor boots", "str_pl": "pairs of faux fur winter survivor boots" },
+    "description": "A pair of customized, Kevlar armored faux fur boots, modified to provide maximum protection from harm and the elements, even when knee-deep in the dead.  While not quite as warm as real fur it's pretty good.",
+    "copy-from": "boots_wsurvivor",
+    "material": [ "kevlar", "faux_fur" ],
```
Comment (actual-nh) [inline data/json/items/armor/boots.json:283]: No adjustment of the warmth value?

Comment (Maleclypse) [reply data/json/items/armor/boots.json:283]: This is a failing on my end but I assumed warmth was tied to material.  Let me know if that's not the case and I'll move this back into draft until I can come back to it.

Comment (actual-nh) [reply data/json/items/armor/boots.json:283]: I have an appointment (video) in 10 minutes so I can't check right now, but IIRC the boots_wsurvivor specify a warmth value, and I don't recall seeing any insulation values for materials (could be nice...).

Comment (Maleclypse) [reply data/json/items/armor/boots.json:283]: Thanks! I'll pull this back for now then.

Comment (actual-nh) [reply data/json/items/armor/boots.json:283]: It is loaded in from JSON, with the default being 0. As far as I can tell, the only way material matters for warmth is wool - anything else doesn't provide as much warmth if wet.

**Comment by Maleclypse** [comment]
@ZhilkinSerg I think this is ready to merge.  Not sure why general tests failed but all the Travis Tests passed.

**Pull request merged**